src/web-component/ay-accordion/index.ts

```diff
@@ -58,21 +48,57 @@ export class AyAccordion extends HTMLElement {
     }
   }
 
+  attributeChangedCallback(name, oldValue, newValue) {
+    if(name === 'disabled') {
+      if(this.hasAttribute('disabled')) {
+        this.querySelectorAll('ay-accordion-header').forEach( (el) => {
+          el.setAttribute('aria-disabled', 'true');
+        });
+      } else {
+        this.querySelectorAll('ay-accordion-header').forEach( (el) => {
+          el.setAttribute('aria-disabled', 'false');
+        });
+      }
+    }
+
+    if(name === 'open') {
+      if(this.open) {
+        Array.prototype.forEach.call(this.children, function (el) {
+          if(!(el.tagName === 'AY-ACCORDION-HEADER')) {
+            el.removeAttribute('hidden');
+          }
+        });
+      } else {
+        Array.prototype.forEach.call(this.children, function(el) {
+          if(!(el.tagName === 'AY-ACCORDION-HEADER')) {
+            el.setAttribute('hidden', '');
+          }
+        });
+      }
+  }
+}
+
   connectedCallback() {
     const childObserver = new MutationObserver(() => {
       Array.prototype.forEach.call(this.children, (el) => this.childCallback(el));
     });
 
     childObserver.observe(this, { childList: true })
 
+    if(this.hasAttribute('open')){
+      this.setAttribute('aria-expanded', 'true');
```

Comment:
It probably makes sense to set `aria-expanded` in the attributeChangedCallback based on the open state?